specification/sbi_atee.adoc

```diff
@@ -13,6 +44,936 @@ running a VMM).
 The following operations must be implemented by the TSM ( _Formal input 
 parameters and return codes are TBD_ ).
 
+== Background
+
+Virtualization platforms are typically comprised of several components including
+platform firmware, host OS, VMM, and the actual payloads that run on them (typically
+in a VM). This model is well established, but the downside is that most platform
+components are in the TCB. This aspect is ill-suited for confidential compute
+workloads (Trusted Execution Environments) that strive to minimize the TCB footprint.
+
+We seek to alleviate this concern by introducing the notion of the a trusted broker for
+TEEs known as the TEE Security Manger (TSM). The idea is that the TSM has a minimal possible
+footprint, and acts as a trusted intermediary between the the untrusted platform components,
+and the TEE workloads that run on it. This is accomplished by having a clear seperation between
+the non-confidential and confidential portions of the workloads running on the platform. The exact
+hardware and software mechanisms used to guarantee the isolation are platform dependent, but
+confidential workloads have absolute execution, data integrity, and confidentiality protections
+from amy component in the non-confidential world. The responsibility of the TSM is to enforce 
```

Comment:
in the above text suggest clarify and restate as - "This is accomplished by the TCB (which includes the TSM and HW) to enforce strict confidentiality and integrity security properties for code execution and data integrity for confidential workloads. The TCB isolates a confidential workload from non-confidential workloads as well as other confidential workloads on the same platform. The exact hardware....".

also I would remove the superlatives like "absolute "

typo "amy" - clarify SW component, and some HW adversary components
from amy component in the non-confidential world